phpBB/phpbb/plupload/plupload.php

```diff
@@ -326,7 +326,7 @@ protected function integrate_uploaded_file($form_name, $chunk, $file_path)
 
 		$tmp_file = $this->temporary_filepath($upload['tmp_name']);
 
-		if (!move_uploaded_file($upload['tmp_name'], $tmp_file))
+		if (!is_writable($this->temporary_directory) || !move_uploaded_file($upload['tmp_name'], $tmp_file))
```

Comment:
phpbb_is_writable would might be better as PHP's is_writable is buggy on windows